src/components/DailyTaskList.js

```diff
@@ -0,0 +1,18 @@
+export default function DailyTaskList() {
+  const dailyTask = dailyTasks.map(dailyTask => <li key={dailyTask.toString()}>{dailyTask} </li>);
+  return <ul>{dailyTask}</ul>;
+}
```

Comment:
Why don't you map over dailyTasks inside the return statement? You don't need the const `dailyTask` anywhere else in this component, so there is no need to outsource it from the return.